tests/gateways/test_jlap.py

```diff
@@ -367,11 +367,12 @@ def warning(*args, **kwargs):
 def test_jlap_flag(use_jlap):
```

Comment:
Looks like we should change the parametrization such that `use_jlap` is a boolean